mesheryctl/internal/cli/root/perf/perf.go

```diff
@@ -69,7 +69,13 @@ mesheryctl perf result -o yaml
 			return cmd.Help()
 		}
 		if ok := utils.IsValidSubcommand(availableSubcommands, args[0]); !ok {
-			return errors.New(utils.PerfError(fmt.Sprintf("'%s' is a invalid command.  Use 'mesheryctl perf --help' to display usage guide.'\n", args[0])))
+			availableSubCmds := []string{"apply", "profile", "result"}
+
+			suggestedCmd := utils.FindClosestArg(args[0], availableSubCmds)
+			if suggestedCmd != "" && suggestedCmd[0] == args[0][0] {
```

Comment:
Could you explain why args[0][0] is used here @gurpreet-legend just curious....

Reply:
Yeah sure,
So, whenever I was trying to find the minimum levenshtein distance between the current sub commands and the one provided by the user, it will always give me an answer to the closest sub command possible out of the valid sub commands. 

To give you an example, let's say the user  entered : `./mesheryctl perf xyz`

Even then the closest possible valid subcommand will be returned by the function. So, what I did here is, I'm checking the initial character of the provided sub command by the user and the one suggested by the function, if they match only then we will suggest a sub command otherwise we'll prompt the user to enter `./mesheryctl perf --help`

Reply:
I see. Impressive solution 👏